modules/installation-vsphere-infrastructure.adoc

```diff
@@ -48,5 +53,13 @@ You can host the VMware vSphere infrastructure on-premise or on a link:https://c
 
 [IMPORTANT]
 ====
-You must ensure that the time on your ESXi hosts is synchronized before you install {product-title}. See link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-8756D419-A878-4AE0-9183-C6D5A91A8FB1.html[Edit Time Configuration for a Host] in the VMware documentation.
+To ensure the best performance conditions for your cluster workloads that operate on OCI and on the Oracle&#174; Cloud VMware Solution (OCVS) service, ensure volume performance units (VPUs) for your block volume is sized for your workloads. 
+
+The following list provides some guidance in selecting the VPUs needed for specific performance needs:
+
+* Test or proof of concept environment: `100` GB, and `20` to `30` VPUs.
+* Basic environment: `500` GB, and `60` VPUs.
```

Comment:
@racedo "Basic environment" or "Base production environment"?

Reply:
Changed to "Base-production environment for now, unless any objections exist?